render/container.go

```diff
@@ -79,9 +79,19 @@ func (r containerWithImageNameRenderer) Render(rpt report.Report, dct Decorator)
 		if !ok {
 			continue
 		}
+		imageName, ok := image.Latest.Lookup(docker.ImageName)
+		if !ok {
+			continue
+		}
+		imageNameWithoutVersion := docker.ImageNameWithoutVersion(imageName)
+		imageNodeID := report.MakeContainerImageNodeID(imageNameWithoutVersion)
+
 		output := c.Copy()
 		output = propagateLatest(docker.ImageName, image, output)
 		output = propagateLatest(docker.ImageLabelPrefix+"works.weave.role", image, output)
+		output.Parents = output.Parents.
+			Delete(report.ContainerImage).
```

Comment:
This is for compatibility with old probes? Regardless of whether I think it's a good idea to remove the image parents, I think a comment is in order. We should probably keep track of changes done for backwards compatibility so that we can potentially remove them after the support life since they clutter the code.

Also, have you tested this with old probes?